### ISSUE TITLE
Is go-z3 still working now?

### ISSUE THREAD

**Issue by CHZofCSU** [mitchellh/go-z3#6]
Following the instruction, I build Z3 first and then go get this project. However, there is an error saying:
mitchellh\go-z3@v0.0.0-20191228203228-4cbedeba863f\ast.go:4:
./go-z3.h:11:10: fatal error: z3.h: No such file or directory
 #include <z3.h>
          ^~~~~~
Thanks a lot for answering.

**Comment by stevegt** [comment]
Which instructions?  According to the README, "You can't simply `go get` this library", so yeah, that's not supposed to work.

https://github.com/mitchellh/go-z3#installation

As shown at the above link, you should only need to do `git clone` of this repo followed by `make`.  If you're manually building z3 and then doing a `go get`, that's not likely to work because among other things your include path isn't going to be set, which would cause the inability to find z3.h.

**Comment by CHZofCSU** [comment]
Yes. I first manually build z3 on Windows, and go get this repo.
So I follow your way, just git clone this repo and make. 
However, the error still exists as shown in the figure. Then I manually use vs command prompt with command: 1) cd ./vendor/z3/build  2) nmake;
Then, I still can't run the example.go. The IDE said it can't find the file libz3.a.

Thanks for answering. I am just curious that is this repo still working now?  Or  I need to build another go-z3.

![githubQ](https://user-images.githubusercontent.com/44314918/143821016-f800f56f-2533-4ced-8e2f-7c54b6ce5ca4.png)

**Comment by stevegt** [comment]
There are Windows instructions in the README.
